menu/menu.yaml

```diff
@@ -130,7 +130,9 @@
         Title: Migrating off NServiceBus.Azure
     - Title: Azure Cloud Services Host
       Articles:
-      - Url: nservicebus/upgrades/acs-host-6to7
+      - Url: nservicebus/hosting/cloud-services-host/acs-host-7to8
+        Title: Version 7 to 8      
+      - Url: nservicebus/hosting/cloud-services-host/acs-host-6to7
```

Comment:
why was acs-host-6to7 moved? the convention for all the other upgrade guides is that they are co-located

Reply:
Define colocated. They are now colocated together. You mean colocated in the nsb/upgrade folder?

Reply:
Happy to move back if important